styles.css

```diff
@@ -0,0 +1,59 @@
+*{
+    padding: 0;
+    margin: 0;
+    background-image: linear-gradient(to bottom right, black, rgb(42, 0, 81)); 
+}
+#header{
+    text-align: center;
+    background-color: rgb(70, 5, 131);
+    padding: 50px;
+    color: aliceblue;
+    font-weight: bolder;
+    font-size: 40px;
```

Comment:
- use units rem, or em instead of px - read about this
- use hex codes for color instead of names